src/MultiSearchPrompt.php

```diff
@@ -43,15 +43,17 @@ public function __construct(
         public mixed $validate = null,
         public string $hint = '',
     ) {
-        $this->trackTypedValue(submit: false, ignore: fn ($key) => Key::oneOf([Key::SPACE, Key::HOME, Key::END, Key::CTRL_A, Key::CTRL_E], $key) && $this->highlighted !== null);
+        $this->trackTypedValue(submit: false, ignore: fn ($key) => Key::oneOf([Key::SPACE, Key::HOME, Key::END, Key::CTRL_A], $key) && $this->highlighted !== null);
 
         $this->initializeScrolling(null);
 
         $this->on('key', fn ($key) => match ($key) {
             Key::UP, Key::UP_ARROW, Key::SHIFT_TAB => $this->highlightPrevious(count($this->matches), true),
             Key::DOWN, Key::DOWN_ARROW, Key::TAB => $this->highlightNext(count($this->matches), true),
-            Key::oneOf([Key::HOME, Key::CTRL_A], $key) => $this->highlighted !== null ? $this->highlight(0) : null,
-            Key::oneOf([Key::END, Key::CTRL_E], $key) => $this->highlighted !== null ? $this->highlight(count($this->matches()) - 1) : null,
+            Key::CTRL_A => $this->highlighted !== null ? $this->toggleAll() : null,
+            Key::CTRL_E => null,
+            Key::oneOf([Key::HOME], $key) => $this->highlighted !== null ? $this->highlight(0) : null,
+            Key::oneOf([Key::END], $key) => $this->highlighted !== null ? $this->highlight(count($this->matches()) - 1) : null,
```

Comment:
This might be intentional but wanted to mention it in case it's not...

When I tried to simplify this to `Key::HOME => ...`, the keybinding wouldn't work at all and the "supports the home and end keys while navigating options" test would fail, which is why I've left this in `Key::oneOf()`.

Reply:
There are multiple escape sequences for the `Home` and `End` keys for various terminals, so their constants are arrays rather than strings:

```php
const HOME = ["\e[1~", "\eOH", "\e[H", "\e[7~"];

const END = ["\e[4~", "\eOF", "\e[F", "\e[8~"];
```

Reply:
I see... that makes sense!